alloccli/alloc.py

```diff
@@ -366,9 +365,9 @@ def search_for_project(self, projectName, personID=None, die=True):
                 self.die("Found more than one project matching: %s" %
                          projectName)
             elif len(projects) > 1 and not die:
-                return projects.keys()
+                return list(projects.keys())
```

Comment:
I can't figure out how to test these ones. What I do know is using `list()` is the safe answer. From what I can figure out it also seems likely that using `list()` is the right answer for these under python3.